GVFS/GVFS.FunctionalTests/Tests/FastFetchTests.cs

```diff
@@ -26,7 +26,7 @@ public class FastFetchTests
         public void InitControlRepo()
         {
             Directory.CreateDirectory(this.fastFetchControlRoot);
-            GitProcess.Invoke("C:\\", "clone -b " + Settings.Default.Commitish + " " + GVFSTestConfig.RepoToClone + " " + this.fastFetchControlRoot);
+            GitProcess.Invoke(this.fastFetchControlRoot, "clone -b " + Settings.Default.Commitish + " " + GVFSTestConfig.RepoToClone + " " + this.fastFetchControlRoot);

```

Comment:
Will this call fail if `this.fastFetchControlRoot` doesn't exist?  The first parameter is `executionWorkingDirectory` and so I'm wondering if we should use a different path (e.g. the parent of `this.fastFetchControlRoot`).